### ISSUE TITLE
[bug] check that berserking works correctly

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#545]
old note:

> berserking is also iffy (control sometimes returns, odd handling of the IE_STATE_ID states - do we do winded after the fact?)

Are we on par how it practically worked in the original?

**Comment by caillean7** [comment]
I compared Minsc's Berserking to the original, and it looks good. Control goes as he is confronted with enemies, then returns several seconds (varies) after all enemies are dead. Attacking party members while berserk, and the bonuses he gets, it all looks the same on GemRB. He is not supposed to get Fatigue after berserking.

Barbarian Rage is fine as well.

I found issues with Enrage (ability for the fighter kit Berserker), though: The Thac0 bonus of -2 is not applied during the enraged state, rather you give a penalty. Example: Fighter with a base Thac0 of 14, actual Thac0 of 13 ends up with 15 / 14 on GemRB, IE gives 14 / 11.

Furthermore, after Enrage wears off, the character should suffer a +2 penalty to Thac0, GemRB gives only +1. Char from the above example has after being enraged on IE 16 / 15, but 15 / 14 on GemRB. The other penalties are applied okay. I don't know if those penalty are already what constitutes 'Fatigue', but on IE the char's portrait gets the Fatigue icon displayed, and it lists it on the character records page. GemRB doesn't do this.
